File: ICPC 2023/Moin Vai OCT 10 2023/a.cpp
//prince

#include<bits\stdc++.h>
using namespace std;

int main()
{
    int t;
    cin>>t;
    while(t--)
    {
        int n;
        cin>>n;
        int a,b,i,count=0;
        for(i=0; i<n; i++)
        {
            cin>>a>>b;
            if(a>b)
            {
                count++;
            }
        }
        cout<<count<<endl;
    }

    return 0;
}

File: ICPC 2023/Moin Vai OCT 10 2023/a.py
# prince

t = int(input())

while t > 0:
    n = int(input())
    count = 0
    i = 0
    while i < n:
        a, b = map(int, input().split())
        if a > b:
            count += 1
        i += 1

    print(count)

    t -= 1
